packages/default/scss/input/_layout.scss

```diff
@@ -430,7 +430,7 @@ $checkbox-size: $icon-size !default;
     .k-form {
         input,
         label:not(.k-checkbox-label):not(.k-radio-label),
-        .k-widget:not(.k-calendar) {
+        .k-widget:not(.k-calendar):not(.k-switch) {
```

Comment:
Do we have another option of selecting this element, without using `:not`?

Reply:
The only possible other solution is to make this selector deeper 
https://github.com/telerik/kendo-themes/blob/develop/packages/default/scss/switch/_layout.scss#L10
However, this will conflict with the approach of styling our widgets(using `.k-widget.k-switch` for this one.

Having said that, using the `:not` selector seems like a better solution in this case.